Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -752,11 +757,10 @@ public function __call($method, $args)
         return call_user_func_array([$this, 'ldapSearch'], $args);
     }
 
-
     /**
      * Check the ldap status code and throw exception on error
      *
-     * @return void
```

Comment:
OK. As it's a subjective preference I should honor your decision as main author:) Commit on the way.